### ISSUE TITLE
Keyboard shortcuts

### ISSUE THREAD

**Issue by JayBrown** [Zeldinator/AudioHotkeys#1]
I was trying to build the app, but I couldn't find any support for system-wide keyboard shortcuts. Are they implemented? (I'm asking because there's no Preferences window/menu.)

**Comment by JayBrown** [comment]
Stupid me. Found them. Modifier 1835008 (CTRL-OPT-CMD) + arrow keys. Sorry.

**Comment by rolbk** [comment]
You're welcome :)

**Comment by JayBrown** [comment]
fyi

Builds worked fine (targets 10.9 & 10.15 x86-64). Auto-conversion/fixing for Swift 5 worked, except one warning (not an error) regarding line 105 in MenuController.swift: `./AudioHotkeys/AudioHotkeys/MenuController.swift:105:20: 'image' was deprecated in macOS 10.14: Use the receiver's button.image instead`

Also made a test build for macOS 11.1 as a Universal 2 binary, but can't test it. (I'm on Intel, *and* still on Catalina.)

As for the MASShortcuts.framework: I tried building it as a universal2, but failed, so I just copied a universal2 binary of the framework from a pre-existing MAS app, and dropped it into AudioHotkeys' copy of the framework… that apparently worked just fine. ;)
